src/interfaces/uproxy_core_api.ts

```diff
@@ -138,6 +144,7 @@ export enum Command {
   VERIFY_USER = 1033,
   VERIFY_USER_SAS = 1034,
   GET_PORT_CONTROL_SUPPORT = 1035,
+  UPDATE_GLOBAL_SETTING = 1036,
```

Comment:
A comment here describing the payload would be great (though I see now literally no other value here specifies it, either).